src/models/Followers.js

```diff
@@ -57,4 +57,18 @@ export default class Followers extends Model {
       onDelete: 'CASCADE'
     });
   }
+
+  /** Returns an array of authors the user is following
```

Comment:
Good job Gilbert. However, can you please align the JSDoc comment to meet with the standard set in the team convention documentation. Thanks

Reply:
I see that , 

> Good job Gilbert. However, can you please align the JSDoc comment to meet with the standard set in the team convention documentation. Thanks

I see that @kennyedward, I wonder why my Eslint didn't flag it.
Thanks